kubectl-rook-ceph.sh

```diff
@@ -465,6 +678,8 @@ function run_start_debug() {
     spec:
         $deployment_spec
 EOF
+    echo "ensure the debug deployment $deployment_name is scaled up"
+    KUBECTL_NS_CLUSTER scale deployments "$deployment_name-debug" --replicas=1
```

Comment:
Is this needed for any case?

Reply:
This is needed because if the original daemon was scaled down, the debug pod needs to be scaled up. In the case of the mon restore, I'm scaling down the mon deployments before starting the debug pod, so I had to scale it back up at least for this scenario. But in other scenarios, this would just be a no-op.

Reply:
Okay I see it is because the deployment spec which is copied, should have the replicas count zero